src/pages/AddressPage/index.tsx

```diff
@@ -348,13 +349,18 @@ export function AddressPage() {
             </Tab>
           ) : null}
 
-          {(type === "erc721" || type === "erc1155") && inventory.length ? (
+          {(type === "erc721" || type === "erc1155") && (
             <Tab
-              title={<Text size="small">Inventory ({inventory.length})</Text>}
+              disabled={!!!inventory.length}
+              title={
+                <TextLoader isLoading={!!!inventory.length} size="small">
+                  Inventory{!!inventory.length ? ` (${inventory.length})`: ''}
+                </TextLoader>
+              }
             >
-              <Inventory inventory={inventory} />
+                <Inventory inventory={inventory} />
             </Tab>
-          ) : null}
+          )}
```

Comment:
A couple of things here:
- the optional rendering is great instead of ternary.
- why triple bang the inventory.length? Could we not use optional chaining + the falsy value of length being 0? I don't understand why triple bang would be necessary here. So just some clarification on this choice or optimize it to being less complex.